core/network/src/error.rs

```diff
@@ -28,13 +28,16 @@ pub enum Error {
 	Io(std::io::Error),
 	/// Client error
 	Client(client::error::Error),
+	/// Some other error occurred.
+	Other(String),
```

Comment:
Why not `DuplicateBootnodeAddress` instead of `Other`?

Reply:
Renamed